client/my-sites/stats/stats-list/stats-list-item.jsx

```diff
@@ -213,15 +214,24 @@ class StatsListItem extends Component {
 		const wrapperClassSet = classNames( {
 			'module-content-list-item-label-section': labelData.length > 1,
 		} );
-
 		const label = labelData.map( function ( labelItem, i ) {
 			const iconClassSetOptions = { avatar: true };
 			let icon;
 			let gridiconSpan;
 			let itemLabel;
 
 			if ( labelItem.labelIcon ) {
-				gridiconSpan = <Gridicon icon={ labelItem.labelIcon } />;
+				switch ( labelItem.labelIcon ) {
+					case 'tag':
+						gridiconSpan = <Icon className="stats-icon" icon={ tag } size={ 22 } />;
+						break;
+					case 'folder':
+						gridiconSpan = <Icon className="stats-icon" icon={ file } size={ 22 } />;
+						break;
+					default:
+						// fallback to an old icon
+						gridiconSpan = <Gridicon icon={ labelItem.labelIcon } />;
```

Comment:
I kept the previous version as a fallback because I couldn't verify if the above two are the only icons used.